cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -128,17 +128,18 @@ private boolean isKafkaAssignerMode(List<String> goals) {
    * Decommission a broker.
    *
    * @param brokerIds The brokers to decommission.
-   * @param dryRun throw
-   * @param throttleDecommissionedBroker whether throttle the brokers that are being decommissioned.
-   * @param goals the goals to be met when decommissioning the brokers. When empty all goals will be used.
+   * @param dryRun Whether it is a dry run or not.
+   * @param throttleDecommissionedBroker Whether throttle the brokers that are being decommissioned.
+   * @param goals The goals to be met when decommissioning the brokers. When empty all goals will be used.
    * @param requirements The cluster model completeness requirements.
-   * @param operationProgress the progress to report.
+   * @param operationProgress The progress to report.
    * @param allowCapacityEstimation Allow capacity estimation in cluster model if the requested broker capacity is unavailable.
    * @param concurrentPartitionMovements The maximum number of concurrent partition movements per broker
    *                                     (if null, use num.concurrent.partition.movements.per.broker).
    * @param concurrentLeaderMovements The maximum number of concurrent leader movements
    *                                  (if null, use num.concurrent.leader.movements).
-   * @return the optimization result.
+   * @param skipHardGoalCheck Whether check and add all hard goals to {@code goals}.
```

Comment:
If we rephrase it as follows, would it be more clear?
`Whether check and add all hard goals to {@code goals}.` -> `True if the provided {@code goals} do not have to contain all hard goals, false otherwise`